scripts/playbooks.py

```diff
@@ -1,37 +1,28 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+import argparse
+import os
 from pathlib import Path
+from typing import Optional
 
-import click
 import networkx as nx
 
-from tdp.cli.utils import collections
+from tdp.core.collection import Collection
+from tdp.core.collections import Collections
 from tdp.core.dag import DEFAULT_SERVICE_PRIORITY, SERVICE_PRIORITY, Dag
 from tdp.core.operation import Operation
 
+TDP_COLLECTION_PATH = os.environ.get("TDP_COLLECTION_PATH")
 
-# TODO: Transform this to a script as it is not really a command (see #346).
-@click.command(
-    short_help="Generate meta playbooks in order to use a TDP like collection without tdp-lib"
-)
-@click.argument("services", required=False, nargs=-1)
-@click.option(
-    "--output-dir",
-    type=Path,
-    help="Output dir where playbooks will be written",
-    required=False,
-    default=".",
-)
-@click.option(
-    "--for-collection",
-    type=str,
-    help="Only write operation from this collection",
-    required=False,
-    multiple=True,
-)
-@collections
-def playbooks(services, output_dir, for_collection, collections):
+
+def playbooks(
+    collections: Collections,
```

Comment:
Add a condition before calling playbooks if you don't want `collections` to be empty.